src/HW_models/NRF_GPIO.c

```diff
@@ -0,0 +1,7 @@
+#include "NRF_GPIO.h"
```

Comment:
Please add license + copyright headers in all new files (HW models as Apache2. For the hal files, just take the BSD header from another of the hal headers)